System.Compression.Primitives/System/IO/Compression/Brotli/BrotliStream.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.IO;
+
+namespace System.IO.Compression
+{
+    public class BrotliStream : Stream
```

Comment:
Stream based APIs should not be in .Primitives components. We use .Primitives suffix for non-allocating APIs that operate on buffers usually represented as Span<byte>. Stream-based APis should be built on top of the primitive APIs and reside in a separate package.